lib/handlers/PatchHandler.ts

```diff
@@ -30,13 +30,13 @@ export default class PatchHandler extends BaseHandler {
       throw ERRORS.INVALID_CONTENT_TYPE
     }
 
-    const file = await this.store.getUpload(id)
+    const upload = await this.store.getUpload(id)
 
     // If a Client does attempt to resume an upload which has since
     // been removed by the Server, the Server SHOULD respond with the
     // with the 404 Not Found or 410 Gone status. The latter one SHOULD
     // be used if the Server is keeping track of expired uploads.
-    const creation = file.creation_date ? new Date(file.creation_date) : new Date()
+    const creation = upload.creation_date ? new Date(upload.creation_date) : new Date()
     const expiration = new Date(creation.getTime() + this.store.getExpiration())
     const now = new Date()
```

Comment:
Creating `Date` object is quite expensive.
```suggestion
    const now = Date.now()
    const creation = upload.creation_date ? new Date(upload.creation_date).getTime() : now
    const expiration = creation + this.store.getExpiration()
```